### DISCUSSION TITLE
Conflict in installing cQube in Azure server

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#826] [General]
Hi, We are facing issues during installation of cQube version release-v5.0.3, in Azure Server - screenshot attached.
Kindly provide us the steps to install cQube in Azure server. Thanks.

<img width="590" alt="screenshot" src="https://github.com/Sunbird-cQube/community/assets/130358614/c2636bba-8c3a-431a-ab2b-d6252f6773f9">

**Comment by bommalla** [comment]
Please follow the below steps one by one. 

sudo docker stop $(sudo docker ps -aq)
sudo docker rm $(sudo docker ps -aq)
sudo docker rmi $(sudo docker images)
sudo docker volume prune
sudo docker network prune
sudo rm -rf ~/cqube-devops

After that freshly clone the repo again then do the below steps:
1. git checkout release v5.0.3
2. git pull
3. git branch
4. apt-get update
5. apt-get upgrade
6. sudo ./install.sh

**Reply by RNarmatha** [reply]
Hi @Mohanapriya777, Please follow the commands which was given by Rajesh and If you facing the same issue please the screenshot for the "Git status " command output.

**Reply by Mohanapriya777** [reply]
Ok

**Comment by bommalla** [comment]
Don't change any code in the config files.just follow the installation document and do the installation.

**Comment by RNarmatha** [comment]
Issue got fixed
